src/Components/Pages/TransferPage.tsx

```diff
@@ -48,9 +51,55 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
   createStyles({
     root: {
       fontFamily: styles.primaryFont,
-      padding: constants.generalUnit * 3,
+      padding: constants.generalUnit * 2.5,
       position: "relative",
       backgroundColor: "white",
+      minWidth: 460,
+    },
+    wrapper: {
+      position: "relative",
+      background: "#FFFFFF",
```

Comment:
Apply the formatting agreements we've agreed with you on in prev PRs.

Reply:
Refactored